KNOWN_ISSUES.md

```diff
@@ -0,0 +1,117 @@
+# Known issues
+
+## Missing metrics 
+
+**Symptoms**
+
+Running
+```
+kubectl top pods
+```
+will return empty result for pods older then 1 minute. e.g.
+```
+W1106 20:50:15.238493 73592 top_pod.go:265] Metrics not available for pod default/example, age: 22h29m11.238478174s
+error: Metrics not available for pod default/example, age: 22h29m11.238478174s
+```
+
+**Debugging**
+
+Please check if your Kubelet is correctly returning pod metrics. You can do that by checking Summary API on any node in your cluster:
+```console
+NODE_NAME=<Name of node in your cluster>
+kubectl get --raw /api/v1/nodes/$NODE_NAME/proxy/stats/summary
+```
+
+This will return JSON that will have two keys `node` and `pods`. 
+Empty list of pods is means that problem is related to Kubelet and not to Metrics Server.
+
+One liner for number of pod metrics reported by first node in cluster (requires [jq](https://stedolan.github.io/jq/)):
+```console
+kubectl get --raw /api/v1/nodes/$(kubectl get nodes -o json  | jq -r '.items[0].metadata.name')/proxy/stats/summary | jq '.pods | length'
+```
+
+**Known causes**
+
+* [Kubelet doesn't report pod metrics in Kubernetes 1.19.0 with Docker <v19](https://github.com/kubernetes/kubernetes/issues/94281)
+  
+  Workaround: upgrade Docker to 1.19+
+
+## Incorrectly configured front-proxy certificate
```

Comment:
I found that the previous formatting made the text more readable, what do you think about applying it to all the issues?

Reply:
By previous do you mean:
* formatting above
* formatting before this PR
?

Reply:
I meant the formatting above